frontend/packages/console-app/src/plugin.tsx

```diff
@@ -11,8 +12,9 @@ const plugin: Plugin<ConsumedExtensions> = [
       id: 'admin',
       name: 'Administrator',
       icon: <CogsIcon />,
-      landingPageURL: '/k8s/cluster/projects',
-      k8sLandingPageURL: '/cluster-status',
+      getLandingPageURL: (flags) =>
+        flags[FLAGS.CAN_LIST_NS] ? '/dashboards' : '/k8s/cluster/projects',
+      getK8sLandingPageURL: () => '/dashboards',
```

Comment:
This won't work in native k8s since it relies on openshift-monitoring prometheus. We probably want to just use namespaces.